static/app/gettingStartedDocs/javascript/angular.tsx

```diff
@@ -293,9 +302,47 @@ const replayOnboarding: OnboardingConfig<PlatformOptions> = {
   nextSteps: () => [],
 };
 
+const feedbackOnboarding: OnboardingConfig<PlatformOptions> = {
+  install: (params: Params) => [
+    {
+      type: StepType.INSTALL,
+      description: tct(
+        'For the User Feedback integration to work, you must have the Sentry browser SDK package, or an equivalent framework SDK (e.g. [code:@sentry/react]) installed, minimum version 7.85.0.',
```

Comment:
```suggestion
        'For the User Feedback integration to work, you must have the Sentry browser SDK package, or an equivalent framework SDK (e.g. [code:@sentry/angular]) installed, minimum version 7.85.0.',
```

Reply:
could also be `[codeAngular:@sentry/angular] or [codeIvy:@sentry/angular-ivy]` as the replay snippet above

Reply:
or we could rephrase this sentence... i don't have a strong suggestion for that though, just an option. I was looking at how it's phrased for the react onboarding and it's a bit different, which is fine to be the same or different too.

Reply:
i'm copying what's in the docs, but will incorporate the first suggestion for sure